src/staticanalysis/bot/static_analysis_bot/clang/tidy.py

```diff
@@ -297,11 +302,17 @@ def as_text(self):
         Build the text body published on reporters
         '''
         message = self.message
+        reason = self.reason
+
         if len(message) > 0:
             message = message[0].capitalize() + message[1:]
-        body = '{}: {} [clang-tidy: {}]'.format(
+        if len(reason) > 0:
+            reason = reason[0].capitalize + reason[1:]
```

Comment:
Nit: I don't think you need to capitalize `reason` (which we edit ourselves in `config.yaml`) the same way as `message` (which is generated by `clang-tidy`).

Please don't do that, we'll try to remember to always begin our `reason` texts with a capital letter.